### PR TITLE
Add Prettier and format all code

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue #3340.

## Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// This file is here, in part, so that users whose code editors are
```
Comment (aaemnnosttv) [inline .prettierrc.js:1]: As a JS file, this should also receive our standard file header.

**Comment by tofumatt** [comment]
This ended up needing more adjustments to the `// eslint-disable-line` comments than I first saw, possibly because of recent changes in the code and the few refactoring PRs that have recently landed.

At any rate, this should cover everything and be good to review 🙂

**Review by aaemnnosttv — COMMENTED** [review]
Thanks @tofumatt – not much to comment on here since most of it is automated but left a few small comments.

File: .storybook/fetch-mocks.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ export function fetchMockSaveSettings() {
 				status: 200,
 				body: JSON.stringify( data ),
 			};
-		},
```
[the remark this answers is not in the document]
Comment (tofumatt) [reply .storybook/fetch-mocks.js:48]: Indeed, but some of the `comma-dangle` modifications (like this one) were a bit blunt. It _appears_ Prettier is a bit smarter about commas, adding dangling commas to lists/arrays/objects but not to arguments like this. _I_ think it makes more sense, and you can still see dangling commas elsewhere, so I think it's actually an improvement 🙂

File: .storybook/main.js [hunk 3]
```diff
@@ -17,11 +17,6 @@
...
 
 module.exports = {
-	stories: [
-		'../stories/**/*.stories.js',
-		'../assets/js/**/*.stories.js',
-	],
-	addons: [
-		'@storybook/addon-viewport',
-	],
+	stories: [ '../stories/**/*.stories.js', '../assets/js/**/*.stories.js' ],
```
[the remark this answers is not in the document]
Comment (tofumatt) [reply .storybook/main.js:20]: Yeah, I agree. An instance of human intervention maybe being a bit better… but again, less to think about 🤷🏻‍♂️

**Comment by aaemnnosttv** [comment]
@googlebot I fixed it.

**Review by aaemnnosttv — APPROVED** [review]
Thanks @tofumatt! Here we go... 😄

**Pull request merged**